test/e2e/assessment_runner_test.go

```diff
@@ -363,7 +363,11 @@ func (tc *testCase) run() {
 						t.Fatal(err)
 					}
 					log.Tracef("Test pod running on node %s", nodeName)
-					usedNydusSnapshotter, err := IsPulledWithNydusSnapshotter(ctx, t, client, nodeName)
+					containerId := tc.pod.Status.ContainerStatuses[0].ContainerID
+					// remove prefix "containerd://"
+					containerId = containerId[13:]
```

Comment:
I think using `strings.CutPrefix` is safer: https://pkg.go.dev/strings#CutPrefix.